src/main/java/io/vertx/core/net/SSLEngineOptions.java

```diff
@@ -64,4 +64,11 @@ public SSLEngineOptions setUseWorkerThread(boolean useWorkerThread) {
     this.useWorkerThread = useWorkerThread;
     return this;
   }
+
+  /**
+   * @return whether to use pooled heap buffers
+   */
+  public boolean usePooledHeapBuffers() {
```

Comment:
this should be only on JdkSSLEngineOptions as this is where it makes sense, if that is useful later we will move it to main class

Reply:
also for data object it should be getter/setter, so that should be get/set

Reply:
So It means that I have to perform instanceof on the SSL helper?

Reply:
yes but it is fine